src/app/api/image/route.ts

```diff
@@ -141,3 +150,40 @@ const validateGetImagesRequest = (data: GetImagesRequest): { valid: boolean, err
 
     return { valid: errors.length === 0, errors };
 }
+
+export async function PUT(request: Request) {
+    const supabaseClient = createClient();
+
+    const data = (await request.json()).formData as ImageResponseData;
+    const { id, ...parsedData }: ImageDatabaseResponseData = putRequestDataToDatabaseData(data);
+
+    if (!id) {
+        return new NextResponse("Fehler: ID muss angegeben werden", {
+            status: 400,
+        });
+    }
+
+    // Validate the parsed data
+    const { valid, errors } = validateData(parsedData);
+    if (!valid) {
+        return new NextResponse(errors.join("\n"), {
+            status: 400,
+        });
+    }
+
+    // Perform the update in Supabase
+    const { error } = await supabaseClient
+        .from('image')
+        .update(parsedData)
+        .eq('id', id);
+
+    if (error) {
+        return new NextResponse("Fehler beim Aktualisieren des Bildes", {
+            status: 500,
+        });
+    }
```

Comment:
api fehler sollten als einheitliches json zurückgegeben werden, also nach dem schema: `NextResponse.json({message: "<error>", {status: <code>})`

Und hier bitte auch den fehler in `error.message` in der response mitgeben